app/Presenters/ComponentPresenter.php

```diff
@@ -12,16 +12,38 @@
 namespace CachetHQ\Cachet\Presenters;
 
 use CachetHQ\Cachet\Dates\DateFactory;
+use CachetHQ\Cachet\Models\Component;
 use CachetHQ\Cachet\Presenters\Traits\TimestampsTrait;
+use Illuminate\Contracts\Config\Repository;
 use Illuminate\Contracts\Support\Arrayable;
-use Illuminate\Support\Facades\Config;
 use McCool\LaravelAutoPresenter\BasePresenter;
 
 class ComponentPresenter extends BasePresenter implements Arrayable
 {
     use TimestampsTrait;
 
     /**
+     * This is the config repository instance.
+     *
+     * @var \Illuminate\Contracts\Config\Repository
+     */
+    protected $config;
+
+    /**
+     * Create a new component presenter instance.
+     *
+     * @param \CachetHQ\Cachet\Models\Component       $resource
+     * @param \Illuminate\Contracts\Config\Repository $config
+     *
+     * @return void
+     */
+    public function __construct(Component $resource, Repository $config)
+    {
+        $this->wrappedObject = $resource;
+        $this->config = $config;
```

Comment:
Should stop using config in here completely tbh. It's not the use of facades that bothers me as such, it's the fact they hide usage like this.